ts/components/screens/OperationResultScreenContent.tsx

```diff
@@ -83,6 +85,7 @@ const OperationResultScreenContent = ({
         /* Android fallback because `centerContent` is only an iOS property */
         Platform.OS === "android" && styles.wrapper_android
       ]}
+      style={{ marginTop: headerHeight ? -headerHeight : undefined }}
```

Comment:
Do we prefer to act on `margins` or on `SafeAreaView` edges (e.g. `edges={["bottom"]}`) when we use this component in a screen showing the react-navigation header? What do you think?

Reply:
yess! Done!